clients/src/main/java/org/apache/kafka/common/serialization/Serdes.java

```diff
@@ -274,6 +282,13 @@ static public Serde<UUID> UUID() {
         return new UUIDSerde();
     }
 
+    /**
+     * A serde for nullable {@code Boolean} type.
```

Comment:
The comment seems to give the impression that users can use the serde to represent `null`, `true` and `false`. I'm not sure if that is entirely true, since `null` has a special meaning in Kafka streams records. Maybe it's safer to drop the `nullable` here?

Reply:
Thank you for the remark! What is wrong with having null for Boolean? I've seen that other data types have it and Boolean type also supports it.

Reply:
I checked again, and I guess you don't have to worry too much about my concern. When I read that comment you wrote, I understand "this class turns `true`, `false`, `null` into a byte array", but it actually maps `null` to `null` and the caller has to find a way how to deal with the `null` result. This seems like this is a convention generally in KStreams, so I think the code is fine, I was just worried about giving the wrong impression to as user of this function, but, as I said, probably safe to ignore.